src/test/scala/edu/ie3/mobsim/io/probabilities/TripProbabilitiesSpec.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.mobsim.io.probabilities
+
+import edu.ie3.mobsim.config.MobSimConfig.CsvParams
+import edu.ie3.mobsim.config.MobSimConfig.Mobsim.Input
+import edu.ie3.mobsim.config.MobSimConfig.Mobsim.Input.{Grid, Mobility}
+import edu.ie3.mobsim.utils.PathsAndSources
+import edu.ie3.test.common.UnitSpec
+
+import java.io.File
+import java.nio.file.Paths
+
+class TripProbabilitiesSpec extends UnitSpec {
+
+  "reads trip probabilities from files correctly" in {
+
+    val gridConf = Grid("grid", CsvParams("/some/Path", ","))
+    val mobSimPath =
+      Paths.get(new File(".").getAbsolutePath, "input/mobilitySimulator")
```

Comment:
Works without dependency on the legacy `File` class:

```suggestion
    val mobSimPath = Paths.get("input", "mobilitySimulator")
```